### PR TITLE
Add hooks to save skp file of current frame to the documents directory

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/shell/ios/sky_surface.h [hunk 1]
```diff
@@ -10,3 +10,7 @@
 -(instancetype) initWithShellView:(sky::shell::ShellView *) shellView;
 
 @end
+
+extern "C" {
+void SaveFrameToSkp();
```
Comment (abarth) [inline sky/shell/ios/sky_surface.h]: SaveFrameToSkPicture ?

File: sky/shell/shell_view.cc [hunk 2]
```diff
@@ -63,5 +63,11 @@ void ShellView::StopDartTracing(
                             base::Passed(&producer)));
 }
 
+void ShellView::SaveFrameToSkp(base::FilePath& destination) {
```
Comment (abarth) [inline sky/shell/shell_view.cc]: SaveFrameToSkPicture

**Comment by abarth** [comment]
LGTM

We'll probably want to eventually re-route this call to the GPU thread but we can worry about that once the compositing system is further along.

**Comment by chinmaygarde** [comment]
Renamed

**Pull request merged**